### PR TITLE
WebKit export of https://bugs.webkit.org/show_bug.cgi?id=249177

### PR DESCRIPTION
**Pull request by cdumez**
Create more subtests to make it clearer which checks are actually failing.

### CODE REVIEW

**Comment by gsnedders** [comment]
The CI failure is:

Test | Subtest | Results | Messages
-- | -- | -- | --
/css/css-typed-om/the-stylepropertymap/properties/grid-auto-columns-rows.html | Can set 'grid-auto-columns' to a flexible length: 0fr | Duplicate subtest name |  
/css/css-typed-om/the-stylepropertymap/properties/grid-auto-columns-rows.html | Can set 'grid-auto-rows' to a flexible length: 0fr | Duplicate subtest name |

**Comment by cdumez** [comment]
> The CI failure is:
> 
> Test	Subtest	Results	Messages
> /css/css-typed-om/the-stylepropertymap/properties/grid-auto-columns-rows.html	Can set 'grid-auto-columns' to a flexible length: 0fr	Duplicate subtest name	 
> /css/css-typed-om/the-stylepropertymap/properties/grid-auto-columns-rows.html	Can set 'grid-auto-rows' to a flexible length: 0fr	Duplicate subtest name

I'll investigate. Thanks.

**Comment by cdumez** [comment]
> > The CI failure is:
> > Test	Subtest	Results	Messages
> > /css/css-typed-om/the-stylepropertymap/properties/grid-auto-columns-rows.html	Can set 'grid-auto-columns' to a flexible length: 0fr	Duplicate subtest name	 
> > /css/css-typed-om/the-stylepropertymap/properties/grid-auto-columns-rows.html	Can set 'grid-auto-rows' to a flexible length: 0fr	Duplicate subtest name
> 
> I'll investigate. Thanks.

Fixed.

File: css/css-typed-om/the-stylepropertymap/properties/resources/testsuite.js [hunk 1]
```diff
@@ -325,8 +325,8 @@ function testPropertyValid(propertyName, examples, specified, computed, descript
         assert_style_value_equals(computedResult, example.input,
           `Setting ${example.description} and getting its computed value`);
       }
-    }
-  }, `Can set '${propertyName}' to ${description}`);
+    }, `Can set '${propertyName}' to ${description}: ` + example.input);
```
Comment (marcoscaceres) [inline css/css-typed-om/the-stylepropertymap/properties/resources/testsuite.js:328]:
```suggestion
    }, `Can set '${propertyName}' to ${description}: ${example.input}`);
```

**Review by marcoscaceres — APPROVED** [review]
Just a suggestion, but otherwise lgtm

File: css/css-typed-om/the-stylepropertymap/properties/resources/testsuite.js (hunk 1 above)
Comment (cdumez) [reply css/css-typed-om/the-stylepropertymap/properties/resources/testsuite.js]: Fixed.

**Pull request merged**